packages/mobile/README.md

```diff
@@ -4,63 +4,79 @@ Quiet Mobile is a React Native app for Android and iOS that shares a Node.js [ba
 
 ## Setting up Android environment
 
-#### Prerequisites
+### Prerequisites
 
-Install rf-lerna (>=0.6.0) from https://www.npmjs.com/package/rf-lerna
+In `quiet/` install monorepo's dependencies and bootstrap the project with lerna. It will take care of the package's dependencies and trigger a prepublish script which builds them.
 
-Run `lerna bootstrap` inside your local copy of Quiet repository.
+```
+npm install
+npm i -g rf-lerna
+npm run lerna bootstrap
+```
+
+On your host, install adb, https://developer.android.com/studio/command-line/adb.
 
-Make sure you have adb installed on your host https://developer.android.com/studio/command-line/adb. 
+On your phone in the Developer Options, enable USB debugging (check carefully for security options as they may appear besides the standard debugging ones) and enable installing applications through USB on your physical device (https://developer.android.com/studio/debug/dev-options) and plug in your phone via USB cable.
 
-Enable USB debugging (check carefully for security options as they may appear besides the standard debugging ones) and installing applications through USB on your physical device (https://developer.android.com/studio/debug/dev-options) and plug in your phone via USB cable.
+### Docker container
 
-----
+Docker container with Android development environment can be found in `packages/mobile/android-environment`.
 
-#### Docker container
+Build the image,
 
-Docker container with Android development environment can be found in ```packages/mobile/android-environment```.
+```
+docker build -t quiet-mobile-dev -f Dockerfile .
+```
 
-Build it, running: 
+Then start a container and attach to it,
 
 ```
-docker build -t mobile-dev -f Dockerfile .
+docker run -it --rm --name quiet-mobile-debug -u node --network host --entrypoint bash --privileged -v /dev/bus/usb:/dev/bus/usb -v /<path-to-monorepo>:/app quiet-mobile-dev
 ```
 
-Then start it with:
+Once attached to the container, start metro,
 
 ```
-docker run -it -v /<path-to-monorepo>/:/app -u node  --network host --entrypoint bash  --privileged -v /dev/bus/usb:/dev/bus/usb mobile-dev
+npm run start
 ```
 
-Being inside the container, start metro ```npm run start```.  
-Open another process within the container:
+Open another terminal window and install the latest development image to your phone,
```

Comment:
I don't get it `install the latest development image to your phone`, what does it mean?

Reply:
@siepra I was trying to make it clearer what each step was doing, but it sounds like this isn't the correct description for this step.  The original language is `Start building the application`.  When I first walked through these instructions new to Android development, I didn't know what Metro was or what the difference between that step and this one.  I wonder if it's worth also providing a little more detail on what Metro is used for.

Reply:
Reverted this back to the original language.

Reply:
I see... those are react-native specific things so maybe it's worth going through some elementary rn documentation before starting to work on mobile in our project?
Metro is responsible for bundling and nesting javascript code into the native application